Adding SRW results (#21)
dtcenter/ufs-case-studies: docs/source/2019Lorenzo.rst

    .. code-block:: bash
   
      crontab Lorenzo.crontab     

  .. group-tab:: SRW.v1.0

>     The case runs are initialized at 12z Sep 25, 2019 with 90 hours forecasting. The app uses ``config.sh`` to define the runtime settings. The settings that need to be modified to set up the first cycle, last cycle, forecast length and cycle hour are listed below.

Reviewer: I see for the Lorenzo case, the initialization time for SRW kept consistent with the MRW and GFSv16.0.10, which is desirable. 
The former Michael case and Barry case for SRW have different initialization times compared with MRW, one is two days later, one is one day later. That will make the interpretation of the results complicated. Is there a reason for the different initialization times?

Author: There are only 90 hour boundary conditions, so the model can not run 5 day forecast to do the exactly comparison.

Reviewer: Yes. I meant the initialization time, not the forecast length. I think it's okay with 90 hrs forecast. Similar to Lorenzo, the initialization time should be consistent for Michael and Barrry case. That is my opinion.

Author: For Michael, if we start from Oct. 7, the forecast will almost miss the maximum part.

Reviewer: That is a good point. Let's see if others have opinions on this. @ligiabernardet @climbfuji

Reviewer 2: agree with linlin

Reviewer 3: agree with @panll and @climbfuji

......................

The physics compset of GFS.v16beta shows improvements in simulating the tracks of hurricane Lorenzo compared with MRW_GFSv15p2. This is related to its better handling of the subtropical high location over the N. Atl. Ocean. However, the MRW_GFSv16beta does not show improvements in capturing the maximum wind speed and minimum surface pressure at the vortex center. In GFS.v.16.10, the right-of-track bias of hurricane Lorenzo over the N. Atl. Ocean still exists. The misrepresentation of the position and/or strength of the subtropical ridge is a key element to explain Lorenzo's early recurvature.

> Hurricane track from SRW_RRFSv1alpha is closer to the Best Track compared to SRW_GFSv15p2 and MRW_GFSv15p2. They all do not capture the hurrican intensities.

Reviewer 2: ```suggestion
Hurricane track from SRW_RRFSv1alpha is closer to the Best Track compared to SRW_GFSv15p2 and MRW_GFSv15p2. Neither of them captures the hurricane intensity correctly.
```
